Snippets/Snippets_3/DelayedDelivery/DeferForDateTime.cs

```diff
@@ -0,0 +1,26 @@
+namespace Snippets3.DelayedDelivery
+{
+    using System;
+    using NServiceBus;
+
+    class DeferForDateTime
+    {
+        IBus bus;
+
+        public DeferForDateTime(IBus bus)
+        {
+            this.bus = bus;
```

Comment:
I get compiler warnings if I do that. I wasn't sure if it would break the build so I copied the pattern of another sample (Callbacks)